backend/capellacollab/sessions/routes.py

```diff
@@ -45,6 +45,7 @@
     GetSessionsResponse,
```

Comment:
As a `read-only` user, the request of `read-only` sessions doesn't work. 
![image](https://user-images.githubusercontent.com/23395732/207371002-57b462c5-3e5c-4ee9-9c39-bb99f51e5cad.png)